src/components/Transcriber.tsx

```diff
@@ -1097,13 +1097,12 @@ export function Transcriber(props: IProps) {
   };
 
   const handleAutoTranscribe = (trans: string) => {
-    var transcription = transcriptionRef.current.firstChild.value;
-    if (trans.trimStart().startsWith(transcription)) transcription = '';
-    showTranscription({
-      transcription:
-        transcription + trans.replace(/[0-9]+:[0-9]+.[0-9]+: /g, ''),
-      position: 0,
-    });
+    const cleanTrans = trans.replace(/[0-9]+:[0-9]+.[0-9]+: /g, '');
+    var curTrans: string = transcriptionRef.current?.firstChild?.value ?? '';
+    if (curTrans.includes(cleanTrans)) return;
+    const transcription = curTrans + cleanTrans;
+    showTranscription({ transcription, position: 0 });
+    setTextValue(transcription);
```

Comment:
What was happening with the old code is that transcription = /v 1 and /v 1 was included in the incoming trans, so I removed it from transcription first.  That still needs to happen and this won't do it.

Reply:
incoming trans won't have timestamps in it so that isn't necessary.

Reply:
ok, I removed the logic about time stamps.